index.html

```diff
@@ -33,52 +34,81 @@
   <script src="https://kit.fontawesome.com/d14f08e7fd.js" crossorigin="anonymous"></script>
 
 
-
-
-
   <link rel="stylesheet" href="assets/css/style.css">
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
I don't see us using `jquery.js` related features anywhere. So, let's remove all these script tags from all the pages.

Reply:
done